Tasks/google-play-promote/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -13,8 +13,10 @@
   "loc.input.help.sourceTrack": "The track you wish to promote from.",
   "loc.input.label.destinationTrack": "Destination Track",
   "loc.input.help.destinationTrack": "The track you wish to promote to.",
+  "loc.input.label.rolloutToUserFraction": "Rollout Release",
+  "loc.input.help.rolloutToUserFraction": "Promote the release to a percentage of users. Use the 'Google Play - Increase Rollout' task to increase the rollout within a track.",
   "loc.input.label.userFraction": "Rollout Fraction",
-  "loc.input.help.userFraction": "Specifies the percentage of users to roll out your APK to.",
+  "loc.input.help.userFraction": "The percentage of users to release the specified APK out to for the specified 'Track'. It can be increased later with the 'Google Play - Increase Rollout' task.",
```

Comment:
Use active voice: "The percentage of users the specified APK will be released to for the specified 'Track'"